downloader/downloader.py

```diff
@@ -5,6 +5,17 @@
 import h5py
 import yaml
 
+# Units for a variety of fields that can be downloaded.
+
+units = {'dft_total_energy': 'Eh',
```

Comment:
If we instead use a standard machine-readable format that could be processed by [`pint`](https://pint.readthedocs.io/en/stable/) or [`qcelemental`](http://docs.qcarchive.molssi.org/projects/QCElemental/en/stable/physconst.html), it would be possible to use this field to _automatically_ convert to the desired units with either tool. I think this just involves some small changes to the strings here, such as `Eh` -> `hartree`, and `e a0` to `elementary_charge*[something]`, but I'm not sure what the `[something]` is for Bohr radius. I can do a bit more digging, but this would be clearer to both humans and unit conversion tools.

Reply:
Done.